### PR TITLE
feat: remove all instructions from boilerplate

### PR DESCRIPTION
**Pull request by ShaunSHamilton**
Discussion might be needed for the README.md changes.

Related to: https://github.com/freeCodeCamp/freeCodeCamp/issues/39720 and https://github.com/freeCodeCamp/freeCodeCamp/pull/39791

### CODE REVIEW

File: README.md
Comment (moT01) [inline README.md]:
```suggestion
```

File: README.md
Comment (moT01) [inline README.md]:
```suggestion
```

File: README.md
Comment (moT01) [inline README.md]: How do we want this @RandellDawson? I think we should add how specifically to have it in the issue. I'm fine with this, although I would make the `F` lowercase

File: README.md
Comment (moT01) [inline README.md]:
```suggestion
# **freeCodeCamp**
```

**Review by moT01 — COMMENTED** [review]
I think we only want the link the challenge in the readme. I don't think it's necessary to add any of this information from the readme to the introduction page (it's mostly already there).

File: README.md
Comment (ShaunSHamilton) [reply README.md]: Are you sure about removing this? I would prefer to leave it.

File: README.md
Comment (RandellDawson) [reply README.md]: I don't think the `**FreeCodeCamp**` needs to be present or anything else all honestly.  The only thing present in the file would be:
```md
# [Introduction to Advanced Node and Express Challenges](https://www.freecodecamp.org/learn/quality-assurance/advanced-node-and-express/)
```
That being said, I suppose having `# freeCodeCamp` would be fine if everyone agrees.  However, the repl.it link should definitely be removed.

**Review by RandellDawson — CHANGES_REQUESTED** [review]
In the main freeCodeCamp repo, the linter uses single quotes for strings instead of double quotes, so I think we should make this consistent in the boilerplate code if possible.  I see many instances of changing single quotes to double quotes.  I think those particular changes should be reverted.  Other than that, I think this PR is good.  We just need to wait until the corresponding PR on for `learn` is merged and the changes have been deployed.

@moT01 Regarding the level one `freeCodeCamp` header, if we are going to leave this here, we should update all the boilerplates the same and update the task listed on the issue.  Currently, we say it should only have the level one heading link to the applicable `learn` page.

File: README.md
Comment (moT01) [reply README.md]: I don't really care either way, I do kind of like the freecodecamp in there. But we did kind of decide not to have it already, so maybe we should just do that. It's pretty minor, but if we can all agree on what we want, then we will know what to put on the rest of them.

What do you think @Sky020?

Comment (ShaunSHamilton) [reply README.md]: @moT01 I go back and forth. It serves no use for the Camper, but outsiders could come across a project repo, and benefit.

If majority of the other boilerplate READMEs do not have it, I would remove this one and the rest.

File: README.md
Comment (ShaunSHamilton) [inline README.md]:
```suggestion
```

File: README.md
Comment (ShaunSHamilton) [inline README.md]:
```suggestion
# [Introduction to Advanced Node and Express Challenges](https://www.freecodecamp.org/learn/quality-assurance/advanced-node-and-express/)
```

**Review by ShaunSHamilton — COMMENTED** [review]
Remove heading

**Review by RandellDawson — APPROVED** [review]
LGTM

**Comment by RandellDawson** [comment]
@moT01 If you merge https://github.com/freeCodeCamp/freeCodeCamp/pull/39791, we can merge this PR once those changes hit production.

**Review by moT01 — APPROVED** [review]
LGTM @Sky020 🎉 Thanks for making this project so much better 🎉

**Pull request merged**
